src/App.jsx

```diff
@@ -14,70 +15,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const prepareGoods = goods.map((good, index) => ({
+  name: good,
+  id: index + 1,
```

Comment:
Nice! But it's better to generate unique random id (e.g. you can use uuid for that)
https://www.npmjs.com/package/uuid